src/pattern-library/components/patterns/UsingComponentsPage.js

```diff
@@ -17,57 +17,52 @@ export default function UsingComponentsPage() {
   return (
     <Library.Page
       title="Using components"
-      intro={<p>This guide applies to updated ({'"next"'}) components only.</p>}
+      intro={
+        <>
+          <p>
+            This package provides several categories of components. Which
+            category a component belongs to defines its general role and the
+            kinds of props it takes.
+          </p>
+          <ul>
+            <li>
+              <strong className="font-semibold">
+                Presentational components
+              </strong>{' '}
+              are composable components that encapsulate design patterns and UI
+              behavior
+            </li>
+            <li>
+              <strong className="font-semibold">
+                Base presentational components
+              </strong>{' '}
+              are presentational components that provide more customization
+              flexibility
+            </li>
+            <li>
+              <strong className="font-semibold">Composite components</strong>{' '}
+              combine composable components and behavior to satisfy a particular
+              use case
+            </li>
+            <li>
+              <strong className="font-semibold">Simple components</strong> are
+              opinionated, simple components that apply design patterns
+            </li>
+          </ul>
+        </>
+      }
     >
```

Comment:
Some visual examples of the different component categories might be a nice addition here in future.

Reply:
Hah! Another rathole I went down into on Friday and then backed out because it was feeling too noisy/complex. I can reassess.